legal-api/src/legal_api/models/filing.py

```diff
@@ -179,7 +179,8 @@ class Source(Enum):
             'submitter_roles',
             'tech_correction_json',
             'temp_reg',
-            'transaction_id'
+            'transaction_id',
+            '_sent_to_gazette_date'
```

Comment:
please move up so that is fits in alphabetically like the others.
probably after 164